.flake8

```diff
@@ -2,4 +2,4 @@
 max-line-length = 120
 per-file-ignores =
     tests/data/service_github.py: E501
```

Comment:
you could maybe remove the `per-file-ignores` line while you're here? it's redundant given that `E501` is in the 'extend-ignore' list